app/components/create-todo/index.tsx

```diff
@@ -0,0 +1,54 @@
+'use client'
+
+import { useState } from 'react'
+import { useRouter } from 'next/navigation'
+import { useAtom } from 'jotai'
+import { appLabels } from '../../constants'
+import { todoListAtom } from '../list-item'
+import { v4 as uuid } from 'uuid'
+
+const CreateTodo = () => {
+  const router = useRouter()
+  const [todoValue, setTodoValue] = useState('')
+  const [todoList, setTodoList] = useAtom(todoListAtom)
+
+  const handleNewTodoCancel = () => {
```

Comment:
We can directly say handleCancel and handleCreate as className is already create-todo so better not to repeat

Reply:
Updated both the function names.